### PR TITLE
change: [M3-8593] - Copy updates in Create with CLI modal

### PR DESCRIPTION
**Pull request by cpathipa**
## Description 📝
Highlight the Pull Request's context and intentions.

## Changes  🔄
List any change relevant to the reviewer.
- The "SDK's" tab is corrected to read "SDKs". 
- The "Getting started" header is corrected to read "Getting Started".

## Target release date 🗓️
9/30

### Verification steps
(How to verify changes)
- Navigate to API CLI Modal in linode create flow
- Verify SDK tab and resource headings. 

## As an Author I have considered 🤔

*Check all that apply*

- [ ] 👀 Doing a self review
- [ ] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ ] 🤏 Splitting feature into small PRs
- [ ] ➕ Adding a [changeset](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md#writing-a-changeset)
- [ ] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [ ] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

File: packages/manager/src/features/Linodes/LinodesCreate/ApiAwarenessModal/ApiAwarenessModal.tsx [hunk 1]
```diff
@@ -88,11 +88,11 @@ export const ApiAwarenessModal = (props: ApiAwarenessModalProps) => {
     sendApiAwarenessClickEvent(`${type} Tab`, type);
 
     const trackingKey =
```
Comment (mjac0bs) [inline packages/manager/src/features/Linodes/LinodesCreate/ApiAwarenessModal/ApiAwarenessModal.tsx:90]: Is this change going to have an impact on LD metrics? (I'm guessing we'll just be able to aggregate the numbers from pre and post typo fix?)

**Review by mjac0bs — APPROVED** [review]
Thanks Chandra! Copy looks good.

Screenshot 2024-09-17 at 10 04 10 AM

**Review by coliu-akamai — APPROVED** [review]
thanks Chandra! approving pending Mariah's comments are resolved/answered 🎉

**Comment by mjac0bs** [comment]
@cpathipa There are two Cypress failures on this branch, and at least one is probably legit, because I'm sure we were testing the previous copy. Can you confirm this is flake or fix:

`linodes/create-linode-view-code-snippet.spec.ts`
`linodes/plan-selection.spec.ts `

**Comment by cpathipa** [comment]
> @cpathipa There are two Cypress failures on this branch, and at least one is probably legit, because I'm sure we were testing the previous copy. Can you confirm this is flake or fix:
> 
> `linodes/create-linode-view-code-snippet.spec.ts` `linodes/plan-selection.spec.ts `

Thanks @mjac0bs updated the create-linode-view-code-snippet.spec file [7ac5957](https://github.com/linode/manager/pull/10954/commits/7ac595722d68d22a30934151c87c4e026ac4460c), And lan-selection.spec passed running in local.

**Pull request merged**